content/basic-concepts/experiment.md

```diff
@@ -4,8 +4,6 @@ match: [experiment, experiments, 'DVC experiments']
 tooltip: >-
   An attempt to reach desired/better/interesting results during data pipelining
   or ML model development. DVC is designed to help [manage
-  experiments](/doc/start/experiment-management), having [built-in
-  mechanisms](/doc/user-guide/experiment-management) like the
-  [run-cache](/doc/user-guide/project-structure/internal-files#run-cache) and
-  the `dvc exp` commands (available on DVC 2.0 and above).
+  experiments](/doc/start/experiment-management), having built-in mechanisms
+  like the [run-cache](/doc/user-guide/pipelines/run-cache)  and `dvc exp` commands.
```

Comment:
```suggestion
  like the [run cache](/doc/user-guide/pipelines/run-cache)  and `dvc exp` commands.
```

I think you are trying to avoid hyphenating `run cache` in the text, right?

Reply:
Yes! Missed this one I guess. Let me double check all instances...

Reply:
Fixed in 0314121ab.